downloads/urls.py

```diff
@@ -9,4 +9,5 @@
     path('release/<slug:release_slug>/', views.DownloadReleaseDetail.as_view(), name='download_release_detail'),
     path('<slug:slug>/', views.DownloadOSList.as_view(), name='download_os_list'),
     path('', views.DownloadHome.as_view(), name='download'),
+    path("feed.rss", views.ReleaseFeed(), name="feed"),
```

Comment:
Would this URL be https://www.python.org/downloads/feed.rss?

Thinking about consistency with our existing feeds, we have these listed in each page's `<head>`:

```html
    <link rel="alternate" type="application/rss+xml" title="Python Enhancement Proposals"
          href="https://peps.python.org/peps.rss">
    <link rel="alternate" type="application/rss+xml" title="Python Job Opportunities"
          href="https://www.python.org/jobs/feed/rss/">
    <link rel="alternate" type="application/rss+xml" title="Python Software Foundation News"
          href="https://feeds.feedburner.com/PythonSoftwareFoundationNews">
    <link rel="alternate" type="application/rss+xml" title="Python Insider"
          href="https://feeds.feedburner.com/PythonInsider">
```

They're all quite different. https://peps.python.org/peps.rss is generated by the https://github.com/python/peps repo. Those feedburner are external. And https://www.python.org/jobs/feed/rss/ is a pretty unusual looking URL for a feed!

So I think it's fine to pick a new scheme for future feeds. There's a lot of variation in feed, see https://blog.jim-nielsen.com/2021/feed-urls/, and something like https://www.python.org/downloads/feed.rss looks good for a feed of data from https://www.python.org/downloads/, and also fits that analysis's conclusions of `mysite.com/{links,posts,notes,podcasts}/feed.{rss,atom,json}`.

---

Let's also add the new feed to the `<head>` for discoverability.

Reply:
> Would this URL be [python.org/downloads/feed.rss](https://www.python.org/downloads/feed.rss)?

Yep!